packages/cli/src/utils/files.js

```diff
@@ -45,16 +45,14 @@ const writeFile = (fileName, data) => {
   return fse.writeFile(fixHome(fileName), data);
 };
 
-// Returns a promise that deletes the file.
-const deleteFile = fileName => {
-  return new Promise(resolve => {
-    try {
-      fse.unlinkSync(fileName);
```

Comment:
I'm not sure why we were wrapping a sync method in a promise (instead of using the async version or no promise at all). Went ahead and did "just plain sync".